lms/templates/discussion/_js_body_dependencies.html

```diff
@@ -1 +1,3 @@
-<%include file="/mathjax_include.html" />
+<%page args="disable_fast_preview=True"/>
```

Comment:
Note to reviewers: even though _js_body_depenencies.html lives in "discussion", it is actually used by several non-discussion related templates. In particular, it is used in courseware.html to load MathJax.

Given that, I had to include the page arg for disabling fast preview here as well as in mathjax_include.html.